### ISSUE TITLE
Add SSL to MarkdownNotes.com

### ISSUE THREAD

**Issue by illera88** [nicbou/markdown-notes#60]
Right now the communication (including the login page) goes over HTTP making it a serious security vulnerability

**Comment by nicbou** [comment]
Yep, it has been a long-standing issue with Markdown Notes. Unfortunately, Markdown Notes has been shelved for a long time, and isn't seeing any new development efforts.
